c_src/TLH_absTab.c

```diff
@@ -0,0 +1,225 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <glib.h>
```

Comment:
:fearful:

This might be NIH syndrome, but I have an aversion to including such a huge library for a single data structure. I'm glad that it (filewatch+glib) works on my machine right now, but I would prefer if we either a) (re)used a self-contained hash table library or b) wrote our own specialized hash table.

@gnuvince, you know the most about our pains with adding 3rd-party libs to rtb-gateway & friends, do you care to weigh in?